homework/style.css

```diff
@@ -1,3 +1,135 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  font-size: 1.5rem;
+  padding: 5%;
+  background-color: pink;
+  color: black;
+}
+
+body {
+  width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #f8f8f8;
+  font-family: 'Roboto', sans-serif;
+  color: rgb(0, 0, 0, 87%);
+  margin-top: 0;
```

Comment:
You are using margin top, left and right, it's easier to use a shorthand; `margin: 0 auto`